crates/core/src/backend/ignore.rs

```diff
@@ -531,29 +566,19 @@ fn map_entry(
     let device_id = if ignore_devid { 0 } else { m.dev() };
     let links = if m.is_dir() { 0 } else { m.nlink() };
 
-    #[cfg(target_os = "openbsd")]
-    let extended_attributes = vec![];
-
-    #[cfg(not(target_os = "openbsd"))]
-    let extended_attributes = {
-        let path = entry.path();
-        xattr::list(path)
-            .map_err(|err| IgnoreErrorKind::ErrorXattr {
-                path: path.to_path_buf(),
-                source: err,
-            })?
-            .map(|name| {
-                Ok(ExtendedAttribute {
-                    name: name.to_string_lossy().to_string(),
-                    value: xattr::get(path, name)
-                        .map_err(|err| IgnoreErrorKind::ErrorXattr {
-                            path: path.to_path_buf(),
-                            source: err,
-                        })?
-                        .unwrap(),
-                })
-            })
-            .collect::<RusticResult<_>>()?
+    let extended_attributes = match list_extended_attributes(entry.path()) {
+        Err(e) => {
+            // TODO - discuss this log:
+            // Most probably that if Xattr read failure happens for one file,
+            // it will also happen for every file in the same subdirectory.
+            //
+            // The end result would be a real flood of errors.
+            // For this reason, I propose to have this log at "debug" level
+            // so it's not transparent, but doesn't impact a "lambda" use case of rustic.
+            debug!("ignoring error: {e}\n");
```

Comment:
So far we throw a warning if there is an error with a specific file - even though this error could come multiple times. So I would also use `warn` here. In future, we might want to collect errors and give some consolidated result at the end, but ATM I think a warning is fine.

Reply:
agreed and applied in [ab9f4ac](https://github.com/rustic-rs/rustic_core/pull/233/commits/ab9f4acdcaf507c9c4f37b59c2a39160433ba252)